docs/handbook/getting-started.md

```diff
@@ -1532,6 +1530,133 @@ Use the following steps to change the root password:
 * Once changed you can exit the su session
 * You should be able to login/authenticate as root now.
 
+## Updating OpenIndiana /dev to /hipster
+
+This section describes how to update your existing OpenIndiana /dev installation to OpenIndiana /hipster.
+
+Note, that although direct updating from /dev to /hipster can be possible, it's not exhaustively tested.
+
+There are several techniques which you can use to update your systems, including root pool installation from OpenIndiana ISO.
+Complete reinstall also can be a decent option.
+
+Further it's considered that you were warned and decided to do more-or-less direct update from /dev to /hipster.
+
+<i class="fa fa-info-circle fa-lg" aria-hidden="true"></i> **NOTE:**
+<div class="well">
+Only server installation update was tested.
+If you do GUI installation update, you are on your own.
+Of course, you are welcome to ask questions in <a href="https://openindiana.org/mailman/listinfo/oi-dev">oi-dev</a> mailing list, but prepare that nobody will guide you through update.
+</div>
+
+* Do backup.
+  <br/>Seriously.
+  The next steps can make your system unbootable.
```

Comment:
From my practice of such update recently, specifically take care about software jumping major versions. Their configurations, data formats, sometimes concepts have changed. Notably, I had to go piece by piece and reconstruct:
* configurations of ISC BIND - can not use same file in different split-views anymore, and other nuances
* data of postgresql - change of on-disk format, must export to SQL backup while old version is available and reimport in new version
* configs of apache - a moving target, especially with dropping of 32-bit this year.

Since in my case most of the services run in local zones, it saved me quite a bit: sometimes I had to clone an older-version's snapshot (e.g. for the postgres case) into a temporary zone, to have the matching set of data and binaries to do the export... which I did not realize I had to do before updating.

Reply:
Because system migration is what is described here, I find warning that "major package updates bring their own set of problems" only optional as it should be obvious from the `pkg update -v` output.